### DISCUSSION TITLE
*PLEASE HELP* Lost Github Pro Status As A Verified Student

### DISCUSSION THREAD

**Discussion by 3N3R4M0** [community/community#44538] [GitHub Education] [answered]
Hello everyone, as you can see from the title, I applied for the GitHub student developer pack and got verified as a student. Everything worked correctly until I got an email from GitHub about a problem with my account billing. When I entered the email I realized that about a month before applying for the GitHub student developer pack, I actually signed up for the 60-day Github pro free trial and completely forgot about it, resulting in getting my account disabled because they couldn't bill my credit card. So I ended up switching to a Free account to reenable my Github account, and I lost my free Github Pro status which was included in the GitHub student developer pack. I tried to reverify my student status but I couldn't because it said that I'm already verified. I was curious if you guys could help me get the Github pro plan again, I would really appreciate it. Thanks.

**Comment by samus-aran** [comment, marked as answer]
Hi @3N3R4M0 👋 

There are a couple of options here to re-instate your Pro subscription as a student. 

1. If this downgrade happened within the past month you should be able to undo the subscription change (via your Billing and plans tab). Look to the bottom of the new subscription plan. It should give you an option to cancel your recent changes and revert your plan back to the student Pro subscription. This only works if the changes haven't exceeded the next billing date. If it has then the other options below would be better.
![billing-testing](https://user-images.githubusercontent.com/4041275/212570739-915bff8c-f78f-4d4c-aad7-db4a78df84c7.png)
2. Open up a[ billing and payments support](https://support.github.com/contact/account?tags=hubberfy_billing_and_payments) ticket and they will be able to re-instate you
3. As you are a verified student if you do [upgrade your personal plan](https://docs.github.com/en/billing/managing-billing-for-your-github-account/upgrading-your-github-subscription) then your student status offsets the costs of having a Pro Subscription.

Hope this all helps and let us know how you get on 👍

**Reply by MrfaridAlam** [reply]
Please help payment and cride Card point

**Comment by 3N3R4M0** [comment]
Hey @samus-aran, thank you so much for answering. I never thought about upgrading to pro because I thought I would get charged for it. As you said, since I'm a verified student, it really did cover the costs of the Pro Subscription. Again, thank you so much for sharing this simple solution with me, have a great day!

**Comment by ARISTEOUROTJR** [comment]
Please help me🙏🙏🙏
